v2.0/backup.md

```diff
@@ -113,15 +113,26 @@ The URL for your backup's destination/locations must use the following format:
 
 `[path to backup]` must be unique for each backup, but the other values depend on where you want to store the backup.
 
+<style>
+	table td:nth-child(2) {
+	    min-width: 130px;
+	}
+</style>
+
 | Backup Location | scheme | host | parameters |
 |-----------------|--------|------|------------|
 | Amazon S3 | `s3` | Bucket name | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` |
 | Azure | `azure` | Container name | `AZURE_ACCOUNT_KEY`, `AZURE_ACCOUNT_NAME` |
 | Google Cloud Storage | `gs` | Bucket name | None––currently only supports instance auth, but we can build non-instance auth at a customer's request |
 | HTTP | `http` | Remote host | N/A |
-| NFS | `nodelocal` | File system location | N/A |
+| NFS [<sup>1</sup>](#considerations) | `nodelocal` | File system location [<sup>2</sup>](#considerations) | N/A |
+| S3-compatible services | `s3` | Bucket name | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_ENDPOINT` |
+
+#### Considerations
 
-{{site.data.alerts.callout_info}}Because CockroachDB is a distributed system, you cannot meaningfully store backups "locally" on nodes. The entire backup file must be stored in a single location, so attempts to store backups locally must point to an NFS drive to be useful.{{site.data.alerts.end}}
+- The backup location parameters often contain special characters that need to be URI-encoded. Use Javascript's [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) function or Go language's [url.QueryEscape](https://golang.org/pkg/net/url/#QueryEscape) function to URI-encode the parameters. Other languages provide similar functions to URI-encode special characters.
+- [<sup>1</sup>](#import-file-temp-directory-urls) Because CockroachDB is a distributed system, you cannot meaningfully store backups "locally" on nodes. The entire backup file must be stored in a single location, so attempts to store backups locally must point to an NFS drive to be useful.
+- [<sup>2</sup>](#import-file-temp-directory-urls) <span class="version-tag">New in v2.0</span> The file system backup location on a local node or NFS drive is relative to the path specified by the `--external-io-dir` flag set while [starting the node](start-a-node.html). If the `--external-io-dir` flag is set to `disabled`, then backups to local directories and NFS drives are disabled. By default, the flag is set to the `extern` subdirectory of the first configured [`store`](#store). To avoid restarting your nodes to set the `--external-io-dir` flag to the locations you want to use, create symlinks to the desired locations from within the `extern` directory.
```

Comment:
- The `<span class="version-tag">New in v2.0</span>` doesn't behave well will bulleted lists. Renders weird:

    <img width="794" alt="screen shot 2018-01-19 at 10 11 27 am" src="https://user-images.githubusercontent.com/15893490/35157158-234d5f8e-fd01-11e7-9cca-1264b37c24e5.png">

    So let's just remove it here. It's called out as new in the `cockroach start` docs, which is sufficient. 

- `(#store)` should be `(start-a-node.html#store)`.

- In the last sentence, "To avoid restarting your nodes to set the `--external-io-dir` flag to the locations you want to use` is confusing me. Can you rephrase?